docs/content/v0.20/examples.md

```diff
@@ -52,17 +52,32 @@ which installs OCM on it.
 
    see [here](./architecture.md#ocm-status-add-on-agent) for more details on the add-on.
 
-5. Create a Workload Description Space `wds1` in KubeFlex. Similarly to before, `-p kubestellar`
+1. Create a Workload Description Space `wds1` in KubeFlex. Similarly to before, `-p kubestellar`
 runs a post-create hook on the *k8s* control plane that starts an instance of a KubeStellar controller
 manager which connects to the `wds1` front-end and the `imbs1` OCM control plane back-end.
 
    ```shell
    kflex create wds1 -p kubestellar
    ```
 
-6. Follow the steps to [create and register two clusters with OCM](example-wecs.md).
+1. Build and run the OCM based transport controller as executable process.  
+**NOTE**: This is work in progress and we're working on running this controller in a pod.
 
-7. (optional) Check relevant deployments and statefulsets running in the hosting cluster. Expect to
+   OCM based transport controller is built in a different package, so we need to download the package and run the executable as background process.
+
+   ```shell
+   OCM_TRANSPORT_PLUGIN_RELEASE="0.1.0-rc2"
+   wget https://github.com/kubestellar/ocm-transport-plugin/archive/refs/tags/v${OCM_TRANSPORT_PLUGIN_RELEASE}.tar.gz
+   tar -xf v${OCM_TRANSPORT_PLUGIN_RELEASE}.tar.gz
+   rm -rf v${OCM_TRANSPORT_PLUGIN_RELEASE}.tar.gz
+   cd ocm-transport-plugin-${OCM_TRANSPORT_PLUGIN_RELEASE}
+   make build
+   ./ocm-transport-plugin --transport-context imbs1 --wds-context wds1 --wds-name wds1 &> transport.log &
```

Comment:
I still not sure why we need to build and not taking the exec ?

Reply:
you are correct, fixed.